src/linker/utilities/general_utils.py

```diff
@@ -103,3 +103,10 @@ def exit_with_validation_error(error_msg: Dict) -> None:
         "\n==========================================\n"
     )
     exit(errno.EINVAL)
+
+
+def merge_dicts(dicts):
+    final_dict = {}
+    for dictionary in dicts:
+        final_dict.update(dictionary)
```

Comment:
This is sensitive to duplicate keynames, i.e. will overwrite. This seems very limited in scope to merging the file bindings that I wonder if it should just be a helper function where it's needed instead of a general util? Don't feel too strongly either way but might be worth at the very least adding a comment that this is a simple wrapper for `.update` and contains all of the caveats when using that method.

Reply:
I was thinking eventually it would be useful for the validation functions where we do {**some_validation, **some_other_validation ...}